main.tex

```diff
@@ -88,6 +88,23 @@
 \end{tikzpicture}}
 % End of drop shadow definition
 
+%Create section heading with graphics. Argument one is heading name, argument two is picture to use on the left. Also adds the section to table of contents.
+\newcommand{\addsection}[2]{
+		\begin{center}
+			\includegraphics[]{\images/sectionheading.png}
+			\vspace*{-20ex}
+			\color{yellow} \Huge \section{#1}
```

Comment:
```suggestion
			\color{yellow} \Huge \section{\uppercase{#1}}
```
I suggest this trick. This will make text all caps in title cards, but normal caps (and more readable IMO) in the content list.

![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/6031c4b9-4dc5-4a1c-9cc5-4722d603ed29)
![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/4ce336e2-5157-4a9e-ac99-e6fbcc9ae25c)

Of course, you need to type "Game Modes" instead of "GAME MODES", etc.